tests/fixups/TraceFixup/main.cpp

```diff
@@ -10,6 +12,14 @@
 
 #include "Config.h"
 
+//////// Need to undefine Preprocessor definition for NONAMELESSUNION on this .cpp file only for this to work
+#define USETRACELOGGING

```

Comment:
This comment is in the wrong spot.

Reply:
Well I'm not sure where the right place is. The change for undefining the preprocessor definition was done in the project file affecting the compilation of this one cpp file only.  Should someone attempt to change that setting (because you don't comment there and it is inconsistent with compilation of other files in the project), this file will fail to compile.  A comment at the top of this file with the keyword that would appear in the compile error would help them out.

Perhaps you have a suggestion for improvement.

Reply:
Why should you not add a comment in the vcxproj file?  If that does not work can you add to the comment where the undefine is?  I didn't even know you could #undefine something in the vcxproj file.

Reply:
I don't normally manually edit the project file. I normally make changes in the project properties from within Visual Studio.  This doesn't display comments, thus I don't expect anyone to see one there!

I guess we all work in different ways!